web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js

```diff
@@ -0,0 +1,39 @@
+import { addConsolidatedProperties } from './addConsolidatedProperties';
+
+describe('addConsolidatedProperties', () => {
+  it('should return the expected object when the message passed in is on the lead case & consolidated group', () => {
+    const result = addConsolidatedProperties({
+      docketNumber: '101-20',
+      leadDocketNumber: '101-20',
+    });
+    expect(result).toMatchObject({
+      consolidatedIconTooltipText: 'Lead case',
+      inConsolidatedGroup: true,
+      inLeadCase: true,
+    });
+  });
+
+  it('should return the expected object when the message passed in is NOT on the lead case & consolidated group', () => {
```

Comment:
nit-pick: Should this test description read something like `should return consolidated case tooltip text, and set inLeadCase to false, when the docket number and lead docket number are different, but in the same consolidated group`

Reply:
changed